File: NonXib/iOS/Weather - Objective C/Weather/ViewController.m
//
//  ViewController.m
//  Weather
//
//  Copyright © 2017 Uber. All rights reserved.
//

#import "ViewController.h"

#import "WeatherService.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    WeatherService *service = [[WeatherService alloc] init];
    GetCitiesResponse *citiesResponse = [service getCities];
    GetWeatherForCityResponse *weatherResponse = [service getWeatherForCity:@"sanfrancisco"];

    NSLog(@"%@", citiesResponse.cities);
    NSLog(@"%@", weatherResponse.currentWeather.longDescription);

}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: NonXib/iOS/Weather - Swift 3/Weather/ViewController.swift
//
//  ViewController.swift
//  Weather
//
//  Copyright © 2017 Uber. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.

        let service = WeatherService()
        let citiesResponse = service.getCities()
        let weatherCityResponse = service.getWeatherForCity("sanfrancisco")

        print(citiesResponse?.cities.first?.name ?? "")
        print(weatherCityResponse?.currentWeather.longDescription ?? "")

    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

